README.md

```diff
@@ -151,6 +161,8 @@ tokenManager: {
 | `redirectUri`  | The url that is redirected to when using `token.getWithRedirect`. This must be pre-registered as part of client registration. If no `redirectUri` is provided, defaults to the current origin. |
 | `authorizeUrl` | Specify a custom authorizeUrl to perform the OIDC flow. Defaults to the issuer plus "/v1/authorize". |
 | `userinfoUrl`  | Specify a custom userinfoUrl. Defaults to the issuer plus "/v1/userinfo". |
+| `ignoreSignature` | Disable ID token signature validation. Defaults to `false`.  |
+| | **Important:** For the Implicit flow, the token signature MUST be validated per [ID token Validation](http://openid.net/specs/openid-connect-core-1_0.html#ImplicitIDTValidation). This option should be used only for browser support and testing purposes. |
```

Comment:
1. Since we're already checking for browser compatibility, we can probably change this to:
    > This option should be used only for testing purposes.
2. We don't have a developer environment for AuthJS, just the Widget. Therefore, we aren't providing a message to the developer that this option isn't recommended (outside of the README).